src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php

```diff
@@ -95,6 +104,7 @@ class="p-5"
 			<?php echo $this->form->renderField('comment'); ?>
 
 		</div>
+		<input type="hidden" id="assigned_user_id" value="<?php echo $this->item->user_id; ?>" />
```

Comment:
same here. this field already rendered at line 67

Reply:
This is hidden field to show the user selected, in ajax we are using this value to show selected and Joomla field id is different from this